.github/workflows/github_action.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   test:
-    name: Running Units Tests
+    name: Running Units Tests and Linting
```

Comment:
Have you considering changing "and" to "&?"

Reply:
Sounds good, I'm on it...